src/frontend/src/btc/components/send/BtcSendForm.svelte

```diff
@@ -0,0 +1,37 @@
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte';
+	import BtcSendAmount from './BtcSendAmount.svelte';
+	import BtcSendDestination from '$btc/components/send/BtcSendDestination.svelte';
+	import type { BtcAmountAssertionError } from '$btc/types/btc-send';
+	import SendForm from '$lib/components/send/SendForm.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import type { NetworkId } from '$lib/types/network';
+	export let networkId: NetworkId | undefined = undefined;
+	export let amount: number | undefined = undefined;
+	export let destination = '';
+	export let source: string;
+
+	let amountError: BtcAmountAssertionError | undefined;
+	let invalidDestination: boolean;
+
+	const dispatch = createEventDispatcher();
+	const close = () => dispatch('icClose');
+</script>
+
+<SendForm {source}>
+	<BtcSendDestination
+		slot="destination"
+		bind:destination
+		bind:invalidDestination
+		{networkId}
+		on:icQRCodeScan
+	/>
+
+	<BtcSendAmount slot="amount" bind:amount bind:amountError />
+
+	<!--	TODO: calculate and display transaction fee	-->
+
+	<button slot="cancel" type="button" class="secondary block flex-1" on:click={close}
```

Comment:
I think, better to leave it as slot. This component will be used either as first step or as second step. In those cases, the "cancel" button changes (`close` or `back`)

Reply:
Oh, I thought we were initially talking only about SendForm. Now, I updated it for BtcSendForm too.

Reply:
shouldn't it be?

`<slot name="cancel" slot="cancel" />`

Reply:
Corrected.